bmorph/evaluation/StatCor_Vis.py

```diff
@@ -0,0 +1,566 @@
+        DataFrames in computations. These will be printed in
+        the legend.
+    uncor_key: the key that accesses the uncorrected data
+        in computations
+    sites: a list or a singular site to be compared in the plot.
+        If multi is set to False and this is not changed to a
+        single site, then the first value in the list will be
+        chosen
+    multi: boolean that determines whether the plot uses data
+        from multiple sites or a single site
+    colors: a list of colors as strings to be plotted from.
+        plotting colors are different for each correction
+        DataFrame, but same across sites for a singular
+        correction. An error will be thrown if there are
+        more cor_keys then colors        
+    
+    """
+    #retreiving DataFrames and establishing data to be plotted
+    datum = computations[datum_key]
+    X = datum - computations[uncor_key]
+    
+    #we need to make the values replacable by the sites,
+    #hence making the max the min and the min the max
+    xmax = X.min().min()
+    xmin = X.max().max()
+        
+    ymax = 0
+    ymin = 100
+    
+    #thrown in are Smax and Smin to determine what the max
+    #and min values overall are so that the axi may be
+    #appropriately scaled
+    fig,ax = plt.subplots()
+    for i,cor_key in enumerate(cor_keys):
+        Y = datum - computations[cor_key]
+        
+            
+        if multi == True:            
+            for site in sites:
+                    x = X.loc[:,site]
+                    y = Y.loc[:,site]
+                    
+                    xmax_site = x.max()
+                    xmin_site = x.min()
+                    ymax_site = y.max()
+                    ymin_site = y.min()
+                    if xmax_site > xmax:
+                        xmax = xmax_site
+                    if xmin_site < xmin:
+                        xmin = xmin_site
+                    if ymax_site > ymax:
+                        ymax = ymax_site
+                    if ymin_site < ymin:
+                        ymin = ymin_site
+                    
+                    scatter_series_axes(x,y,site,colors[i],0.05,ax)
+        else: #meaning site should be set to a singular value
+            #double check that this was actually changed, otherwise picks first value
+            site = sites
+            if type(sites) == list:
+                site = sites[0]
+
+            x = X.loc[:,site]
+            y = Y.loc[:,site]
+            xmax_site = x.max()
+            xmin_site = x.min()
+            ymax_site = y.max()
+            ymin_site = y.min()
+            if xmax_site > xmax:
+                xmax = xmax_site
+            if xmin_site < xmin:
+                xmin = xmin_site
+            if ymax_site > ymax:
+                ymax = ymax_site
+            if ymin_site < ymin:
+                ymin = ymin_site
+            
+            scatter_series_axes(x,y,site,colors[i],0.05,ax)    
```

Comment:
Prepend a `return` to the front of this, with addition of returning `fig, ax` from previous function call.